Flank/flank
# 905 - Print devices screen orientations

Fixes #905 

## Test Plan
> How do we know the code works?

1. Execute ` flank firebase test ios|android test-environment ` should print on bottom screen orientation table

2. Execute ` flank firebase test ios|android screen-orientations list ` should print screen orientation table

3. Execute ` flank ios|android screen-orientations list ` should print screen orientation table

```

┌────────────────┬───────────┬─────────┐
│ ORIENTATION_ID │   NAME    │   TAG   │
├────────────────┼───────────┼─────────┤
│   landscape    │ Landscape │         │
│    portrait    │ Portrait  │ default │
│    default     │  Default  │         │
└────────────────┴───────────┴─────────┘

```

## Checklist

- [X] release_notes.md updated
- [X] Solution for iOS
- [X] Solution for Android
